src/graphics/gfx_update.cpp

```diff
@@ -2630,9 +2631,9 @@ void UpdateGraphics(bool skipRepaint)
 //                    BitBlt myBackBuffer, .X - vScreen(Z).Left, .Y - vScreen(Z).Top, 32, 32, GFX.ECursor(2).hdc, 0, 0, vbSrcPaint
                     XRender::renderTexture(curX, curY, GFX.ECursor[2]);
 //                    If LCase(frmLayers.lstLayer.List(frmLayers.lstLayer.ListIndex)) <> "default" Then
-                    if(!e.Layer.empty() && SDL_strcasecmp(e.Layer.c_str(), "Default") != 0)
+                    if(e.Layer != LAYER_NONE && e.Layer != LAYER_DEFAULT)
 //                        SuperPrint UCase(frmLayers.lstLayer.List(frmLayers.lstLayer.ListIndex)), 3, .X + 28, .Y + 34
-                        SuperPrint(EditorCursor.Layer, 3, curX + 28 , curY + 34);
+                        SuperPrint(Layer[e.Layer].Name, 3, X + 28, Y + 34);
```

Comment:
Are you sure it's valid X and Y instead of `curX` and `curY`? Isn't a mistake?

Reply:
You are right, thank you!